meetings/2021-07-07.md

```diff
@@ -0,0 +1,111 @@
+    * AccessPolicy - https://github.com/solid/authorization-panel/issues/145
+    * AccessPolicyResource - https://github.com/solid/authorization-panel/issues/217
+    * Policy resource in ontology - https://github.com/solid/authorization-panel/issues/142
+* UC1 Resource access - https://github.com/solid/authorization-panel/pull/215
+* UC3 Inheritance - https://github.com/solid/authorization-panel/pull/216
+* Initial use case for consent workflow - https://github.com/solid/authorization-panel/pull/232 & https://github.com/solid/authorization-panel/pull/231
+* Relaxing acl:default - https://github.com/solid/authorization-panel/issues/191
+* Remove discussions from the repository - https://github.com/solid/authorization-panel/issues/226
+      * Part of point below :point_down: 
+* Strategy for processing 90 (and growing) issues currently open
+* ACP Draft status
+
+
+## Present
+
+* Elf Pavlik
+* Eric Pru'Hommeaux
+* Justin Bingham
+* Henry Story
+* Matthieu Bosquet
+* Sarven Capadisli
+
+
+## Minutes
+
+### ACL default
+
+Let's discuss further on GitHub and catch-up on it next week.
+
+### Issues on WAC vs Authorization panel
+
+Sarven: Should we move that issue to the WAC repo.
+
+Henry: But then, should we follow the issues in the WAC repo to discuss them in the authorization panel?
+
+Pavlik: I think that issue should be moved to the WAC repo since it is very specific to WAC.
+
+Henry: What about WAC+?
+
+Sarven: I think high level authorization stuff is ok to be discussed in the authorization panel.
+
+Henry: I'm happy to incubate here and move once there is enough feedback (where there would be more visibility for implementers).
+
+Matthieu: +1 to that
+
+Henry: Is the newest WAC spec finished now?
+
+Sarven: Yes, it is merged and published as an ED. This week I'll make a new PR for the Solid specification. The version that among us should be considered as a working draft kind of document. I don't expect much reviewing to happen. ED is equivalent to a WD. We said that we can't expect changes substantial at any level before we arrive at an implementable spec.
+
+Pavlik: I think I agree with Solid OIDC but I understood that WAC is a special case trying to stay more historic and not have substantial changes.
+
+Sarven: No, or rather it is hard to answer. It does not have major changes to what it is supposed to be as part of the protocol. But it doesn't mean that all the issues and extensions being proposed can't be introduced.
+
+Pavlik: OK, that clarifies my understanding.
+
+Eric: I think the instruction was document as is and carry on.
+
+Sarven: But there are things acknowledged like client ID for example or extended access modes that could be introduced to WAC. I think we'll go case by case to know whether it is part of this version or subsequent versions.
+
+Henry: I think the imports is one feature that could solve a problem and be moved to WAC+ and it might get more reviews if it is on the WAC repository. Could I move it myself?
```

Comment:
```suggestion
Henry: I think the imports is one feature that could solve a problem and be moved to WAC+, and it might get more reviews if it is on the WAC repository. Could I move it myself?
```